pom.xml

```diff
@@ -24,24 +24,71 @@
     <version>7</version>
   </parent>
 
-  <groupId>com.google.common.geometry</groupId>
-  <artifactId>s2-geometry-library-java</artifactId>
-  <version>HEAD-SNAPSHOT</version>
+  <groupId>com.google.geometry</groupId>
+  <artifactId>s2-geometry</artifactId>
+  <version>2.0.0</version>
   <name>S2 Geometry Library for Java</name>
   <description>
     Core geometry libraries for Java.
   </description>
+  <url>https://github.com/google/s2-geometry-library-java</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Chris Povirk</name>
```

Comment:
This is way overstating things, and it would be more accurate to leave me out, but we can still do this for 2.0.0 purposes to match the published pom :)

Reply:
You mirrored 9 years of internal changes, so you more than deserve to be mentioned!